starter-code/src/main.js

```diff
@@ -24,27 +24,115 @@ var cards = [
   { name: 'the avengers',    img: 'the-avengers.jpg' },
   { name: 'thor',            img: 'thor.jpg' }
 ];
-
 $(document).ready(function(){
-  var memoryGame = new MemoryGame(cards);
-  var html = '';
-  memoryGame.cards.forEach(function (pic, index) {
-    html += '<div class= "card" id="card_' + pic.name + '">';
-    html += '<div class="back"';
-    html += '    name="'       + pic.img +  '">';
-    html += '</div>';
-    html += '<div class="front" ';
-    html += 'style="background: url(img/' + pic.img + ') no-repeat">';
-    html += '</div>';
-    html += '</div>';
-  });
+  
+  
+	var placeCards = function(){
+		var html = '';
+		cards.forEach(function (pic, index) {
+			html += '<div class= "card" id="card_' + pic.name + '">';
+			html += '<div class="back"';
+			html += '    name="'       + pic.img +  '">';
+			html += '</div>';
+			html += '<div class="front" ';
+			html += 'style="background: url(img/' + pic.img + ') no-repeat">';
+			html += '</div>';
+			html += '</div>';
+			$('#memory_board').html(html);
+	  });
+	  // Add all the div's to the HTML
+	  document.getElementById('memory_board').innerHTML = html;
+	  
+	  $('.back').on('click', function () {
+			/*console.log($(this).parent().attr('id'));
+			var cardName = $(this).parent().attr('id');
+			console.log($('#' + cardName));
+			$('#' + cardName).eq(0).toggleClass('back');*/
+			
+			$(this).toggle();
+			$(this).parent().find('.front').toggle();
+			
+			myGame.arrComp.push($(this));
+			console.log(myGame.arrComp);
+			
+			if(myGame.arrComp.length == 2){
+				var result = myGame.checkIfPair(myGame.arrComp[0], myGame.arrComp[1]);
+				$('#pairs_clicked').text(myGame.pairsClicked);
+				$('#pairs_guessed').text(myGame.pairsGuessed);
+	
+				
+				if(!result){
+					setTimeout( function() {
+					myGame.arrComp[0].toggle();
+					myGame.arrComp[0].parent().find('.front').toggle();
+					myGame.arrComp[1].toggle();
+					myGame.arrComp[1].parent().find('.front').toggle();
+					myGame.arrComp = [];
```

Comment:
in order to allow for others to then choose a card after your setTimeout function you should do `$('.back').removeClass('blocked');` both at the end of your timeout function and again within your else in order to allow people to choose another pair whether they have a true or false result when checking if you have a pair.